src/polyswarm_api/types/resources.py

```diff
@@ -137,14 +138,9 @@ def valid_assertions(self):
 
     @property
     def filenames(self):
-        if self._filenames is None:
-            for metadata in self.json.get('metadata', []):
-                if metadata.get('tool') == 'scan':
-                    self._filenames = metadata.get('tool_metadata', {}).get('filename', [])
-                    break
-            else:
-                self._filenames = []
-        return self._filenames
+        warnings.warn('This property is deprecated and will be removed in the next major version. '
+                      'Please use "Metadata().filenames" in the future.')
+        return []
```

Comment:
should probably return "Metadata().filenames" here instead of "[ ]" else this isn't backward compatible.

Reply:
This field is already empty for most cases or has outdated information. The list prevents breakage, but I would stop returning outdated information here. If we run the data migration to remove `scan` tools from the db, it will always render an empty list. I initially used `self.metadata.filenames` but realized that is actually inconsistent information we are providing. I can change if you think it is better.

Reply:
ok. this works.